### ISSUE TITLE
selenium.common.exceptions.MoveTargetOutOfBoundsException on mouse_out

### ISSUE THREAD

**Issue by jstray** [cobrateam/splinter#579]
When trying to use mouse_out() on GeckDriver, I get:

```
selenium.common.exceptions.MoveTargetOutOfBoundsException: Message: (5407.5, 5098) is out of bounds of viewport width (1280) and height (701)
```

**Comment by jsfehler** [comment]
@andrewsmedina @jstray This issue should be fixed with the release of 0.11.0

**Comment by andrewsmedina** [comment]
closed in 011.0 release
